Portfolio/index.css

```diff
@@ -0,0 +1,772 @@
+    padding: 6px 12px;
+    background:transparent ;
+    border: 1px solid  #00D8FF;
+}
+.phnbtn  strong{
+    background:transparent;
+    color: #00D8FF;
+    font-family: Roboto;
+ width: 34px;
+    padding: 8px 16px;
+ 
+    /* border: 1px solid #00D8FF; */
+  
+    cursor: pointer;
+}
+
+
+.navbar img {
+    margin-left: 10px;
+ 
+}
+
+
+
+
+.abtme{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #004572;
+  
+    position: relative;
+    height: 500px;
+}
+.abtme img{
+    
+    margin-top: 66px;
+   
+}
+.abtme h1{
+    margin-left: 100px;
+    display: flex;
+    color:  #00D8FF;
+    
+    font-size: 3rem;
+    margin-top: -162px;
+    /* border: 2px solid red; */
+}
+.abtme h2{
+    margin-bottom: -54px;
+    margin-left: 20px;
+    color:  #00D8FF;
+    font-size: 3rem;
+}
+
+/* get to know me */
+.gettoknowme{
+    display: inline-block;
+    height: 300px;
+    background-color: #003E66;
+   
+}
+.gettoknowme h1{
+    color: #00D8FF;
+    margin-top: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.name{
+  margin-left: 298px;
+
+}
+.name2{
+    color: #9ABEE9;
+}
+.gettoknowme p{
+    color: #9ABEE9;
+    margin-top: 34px;
+    width: 60%;
+  margin-left: 300px;
+    display: block;
+
+ 
+}
+
+/* skills */
+
+.myskills{
+    height: 400px;
+    background-color: #004572;
+}
+.myskills h1 {
+    color: #00D8FF;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    line-height: 2;
+    font-weight: 400;
+    font-size: 40px;
+    font-family:  Revalia;
+}
+.myskills2 h1{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 40px;
+    font-weight: 400;
+    font-family: Revalia;
+    color: #00D8FF;
+    line-height: 90px;
+}
+
+.containerss {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: center;
+	min-height: 35vh;
+	padding: 20px;
+	box-sizing: border-box;
+
+}
+
+
+.image-container {
+	width: 250px;
+  
+   
+	max-width: 300px;
+	margin-bottom: 20px;
+}
+
+.image-container img {
+	width: 50px;
+    filter: invert(60%);
+	height: auto;
+}
+
+.text-container {
+    color:  #9ABEE980;
+	text-align: center;
+    font-weight: 500;
+    font-size:  20px;
+}
+
+.image-container h2 {
+	margin-bottom: 10px;
+    color:  #9ABEE980;
+}
+ 
+
+
+
+
+
+
+
+.explore-btn button {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #00D8FF;
+    padding: 4px 15px;
+    color: #003E66;
+    border-radius: 10px;
+    /* margin-left: 29px; */
+    cursor: pointer;
+    background-color: #00D8FF;
+}
+.explore-btn{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-right: 35px;
+    margin-left: 40px;
+    
+    
+}
+.contact {
+    background-color: gray;
+
+}
+
+
+/* projects */
+
+
+.project-section {
+    display: flex;
+    flex-direction: column;
+  }

```

Comment:
Consider adding some space after the closing curly brace for better readability.